samples/csharp/web/avatar/Controllers/AvatarController.cs

```diff
@@ -0,0 +1,756 @@
+﻿using System.Web;
```

Comment:
Add license clarification at the beginning of each source code file, like this:
https://github.com/jinshanzhao/cognitive-services-speech-sdk/blob/master/samples/csharp/sharedcontent/console/Program.cs#L1

Reply:
added